### PR TITLE
[CAS-1036] Add theme attribute to customize FileAttachmentsView

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://stream-io.atlassian.net/browse/CAS-1036

### 🎯 Goal

Allow customization of `FileAttachmentsView` via theme attribute

### 🛠 Implementation details

1. Added `streamUiMessageListFileAttachmentStyle` theme attribute 
2. Declared `StreamUi.MessageList.FileAttachment` style

The naming is slightly different to what is suggested in the ticket.

### 🧪 Testing

Override `streamUiMessageListFileAttachmentStyle` theme attribute:

```xml
<style name="StreamTheme" parent="@style/StreamUiTheme">
    ...
    <item name="streamUiMessageListFileAttachmentStyle">@style/FileAttachmentTheme</item>
</style>
    
<style name="FileAttachmentTheme" parent="StreamUi.MessageList.FileAttachment">
     ...
     <item name="streamUiFileAttachmentTitleTextColor">@color/stream_ui_accent_red</item>
</style>
```

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### 🎉 GIF

giphy (1)

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt [hunk 1]
```diff
@@ -10,11 +10,12 @@ import io.getstream.chat.android.ui.R
 import io.getstream.chat.android.ui.TransformStyle
 import io.getstream.chat.android.ui.common.extensions.internal.dpToPx
 import io.getstream.chat.android.ui.common.extensions.internal.getColorCompat
+import io.getstream.chat.android.ui.common.extensions.internal.getDimension
 import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
 import io.getstream.chat.android.ui.common.extensions.internal.use
 import io.getstream.chat.android.ui.common.style.TextStyle
 
-public data class FileAttachmentsViewStyle(
+public data class FileAttachmentViewStyle(
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt:18]: Decided to do this without deprecation as these changes are recent and evident. In most cases (attrs, transformer, etc.) the singular form is used, therefore changing the name here

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt [hunk 2]
```diff
@@ -26,23 +27,30 @@ public data class FileAttachmentsViewStyle(
...
 ) {
     internal companion object {
-        operator fun invoke(context: Context, attrs: AttributeSet?): FileAttachmentsViewStyle {
-            context.obtainStyledAttributes(attrs, R.styleable.FileAttachmentView).use { attrsArray ->
+        operator fun invoke(context: Context, attrs: AttributeSet?): FileAttachmentViewStyle {
+            context.obtainStyledAttributes(
+                attrs,
+                R.styleable.FileAttachmentView,
+                R.attr.streamUiFileAttachmentViewStyle,
+                R.style.StreamUi_FileAttachmentView
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt:35]: "Attachment" is singular everywhere

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt (hunk 1 above)
Comment (adasiewiczr) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/FileAttachmentViewStyle.kt:18]: Let's just mention this in CHANGELOG and we are good to go

File: stream-chat-android-ui-components/src/main/res/values/attrs_file_attachment_view.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- MediaAttachmentView -->
+    <!-- FileAttachmentsView -->
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/res/values/attrs_file_attachment_view.xml:3]:
```suggestion
    <!-- FileAttachmentView -->
```

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**